### PR TITLE
Use state machines graphviz library to generate diagrams

### LINKED ISSUE #629: Update state machine diagrams
@jcoyne Can I ask you to update the state machine diagrams you created for the inception deck? We've evolved the state machine a bit in code since the original design, and I'm hearing that it'd be valuable to keep these diagrams up to date so we have a visual way to make sense of our state machine flow. Once updated, can you link them in the [(new) dev wiki](https://github.com/sul-dlss/happy-heron/wiki)?

### PR DESCRIPTION
**Pull request by mjgiarlo**
Fixes #629

## Why was this change made?

These state machine images, plus instructions on how/when to regenerate them, will live in our dev docs (wiki): https://github.com/sul-dlss/happy-heron/wiki#state-machine-diagrams. They help us make better sense of our state transitions.

## How was this change tested?

:man_shrugging: 

## Which documentation and/or configurations were updated?

See above

### CODE REVIEW

File: app/services/broadcast_work_change.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# typed: true
+# typed: strict
```
Comment (mjgiarlo) [inline app/services/broadcast_work_change.rb:1]: Sorbet flipped this for me on push.

**Review by jcoyne — CHANGES_REQUESTED** [review]
How about generating svgs rather than PNGs? That would make them more useful at every resolution.

**Review by ndushay — APPROVED** [review]
Thank you for this!

**Review by ndushay — COMMENTED** [review]
why does Collection have a transition "begin deposit"  from deposited to depositing without it going to version draft???

image

**Comment by mjgiarlo** [comment]
@ndushay :speech_balloon: 
> why does Collection have a transition "begin deposit" from deposited to depositing without it going to version draft???
> 
> image

I don't know atm. Maybe the state machine diagrams are already proving worthwhile :laughing:

**Review by jcoyne — APPROVED** [review]

**Pull request merged**